SolarSystemBreakfast_colorize.rb

```diff
@@ -0,0 +1,78 @@
+# Projest Solar System. Breakfast level
+require "colorize"
+
+class Planet
+  attr_accessor :name, :distance, :radius, :mass, :rotation
+
+  def initialize(planets)
+    @name = planets[:name]
+    @distance = planets[:distance]
+    @radius = planets[:radius]
+    @mass = planets[:mass]
+    @rotation = planets[:rotation]
+  end
+
+  def to_s
+   "Planet: #{@name} distance: #{@distance} radius: #{@radius} mass: #{@mass} rotation: #{@rotaion}"
+  end
+end
+
+
+class Solar_system
+attr_reader :planet
+
+  def initialize
+    @planet = {}
+  end
+
+  def add_planet(constant_var_planet)
+    key = constant_var_planet.name
+    @planet[key] = constant_var_planet
+  end
+end
+
+
+# Earth parameters set up to 1.0, as the other planets' parameters are calculated in camparison to the Earth
+# Creating objects of class Planet
+
+earth = Planet.new(name:"Earth".colorize(:green).on_blue, distance: 1.0, radius:1.0, mass:1.00, rotation: 1.0)
```

Comment:
So the one thing to be careful with here is that in your Solar_system contructor, you use the defined string value of `name` as the hash key. But what you're doing here is actually making `name` colorized, which makes the string value look like `"\e[0;32;44mEarth\e[0m"`.

That could lead to odd results, especially if someone expects planets to have non-colorized names, like this:
```
solar_system.planet['Earth']
=> nil
```

And easy way to fix this is to use the `.uncolorize` method from the colorize gem in your `add_planet` method:
```
  key = constant_var_planet.name.uncolorize
```